src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,259 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.exception.DukeException;
+import duke.exception.EmptyDescriptionException;
+import duke.exception.EmptyDateException;
+import duke.exception.WrongCommandException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class Duke {
+
+    static ArrayList <Task> taskArrayList = new ArrayList<Task>();
+    public static String format = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\n";
+    static int count = 0;
+
+    public static void taskAdder(ArrayList<Task> taskArrayList) {//, int count){
+        System.out.println("Got it. I've added this task: \n" + taskArrayList.get(taskArrayList.size() - 1));
+        //count++;
+        System.out.println("Now you have " + taskArrayList.size() + " tasks in the list");
+        //return count;
+    }
+
+    public static void taskRemover(ArrayList<Task> taskArrayList, int index){//, int count){
+        System.out.println("Noted. I've removed this task: \n" + taskArrayList.get(index));
+        //count--;
+        taskArrayList.remove(index);
+        System.out.println("Now you have " + taskArrayList.size() + " tasks in the list");
+        //return count;
+    }
+
+    public static void parseInput(String[] userInput) throws EmptyDescriptionException, WrongCommandException,
```

Comment:
As a general rule, all public methods of a class should have a descriptive header comment